app/controllers/assessments_controller.rb

```diff
@@ -211,16 +211,19 @@ def update
           delta << "#{symptom.label} (\"#{old_val}\" to \"#{new_val}\")"
         end
       end
-
       assessment.symptomatic = assessment.symptomatic?
+      old_reporter = assessment.who_reported
       # Monitorees can't edit their own assessments, so the last person to touch this assessment was current_user
       assessment.who_reported = current_user.email
 
       # Attempt to save and continue; else if failed redirect to index
       return unless assessment.save
 
       comment = 'User updated an existing report (ID: ' + assessment.id.to_s + ').'
-      comment += ' Symptom updates: ' + delta.join(', ') + '.' unless delta.empty?
+      unless delta.empty?
+        comment += ' Symptom updates: ' + delta.join(', ') + '.'
+        comment += " Reporter updated: (\"#{old_reporter}\" to \"#{current_user.email}\")." unless old_reporter == current_user.email
+      end
```

Comment:
is there a reason why this new part of the comment needs to be wrapped in the `unless delta.empty?` condition?

```suggestion
      comment += ' Symptom updates: ' + delta.join(', ') + '.' unless delta.empty?
      comment += " Reporter updated: (\"#{old_reporter}\" to \"#{current_user.email}\")." unless old_reporter == current_user.email
```

Reply:
I believe it's because if we don't update any symptoms, we don't make any changes to the history, and the reporter isn't actually updated. Let me check, though!